gcp/observability/opencensus.go

```diff
@@ -87,7 +89,8 @@ func startOpenCensus(config *config) error {
 		return nil
 	}
 
-	exporter, err := newExporter(config)
+	var err error
+	exporter, err = newExporter(config)
```

Comment:
Is this changed needed? If not, please revert.  If so, maybe we have something else that needs fixing instead.

Reply:
Talked about this offline, this is to write to global.